docs/contributing.rst

```diff
@@ -113,6 +113,21 @@ Documentation is written in `Sphinx-flavored reStructuredText <sphinx:index>`, s
 Feel free to file issues or pull requests as well if you wish something was better documented, as this too can help prioritize.
 
 
+**Building the Documentation Locally**
```

Comment:
If you want another layer of heading (which seems OK), you should use a heading rather than just bolding the text.

The syntax for that is:

```suggestion
Building the Documentation Locally
===========================
```

but I'll apply that change manually and merge.

Thanks again!
```